lms/static/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,105 @@
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    display: inline-block;
+    position: relative;
+
+    .button-more.has-dropdown {
```

Comment:
NIT: correct [property order](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order).